src/main/scala/io/iohk/ethereum/network/PeerActor.scala

```diff
@@ -261,6 +270,18 @@ class PeerActor(
 
       case GetStatus =>
         sender() ! StatusResponse(Handshaked)
+
+      case StartFastSync(targetHash) =>
+        //todo get storage form somewhere?
+        val storage = FastSyncActor.Storage(
+          new BlockHeadersStorage(EphemDataSource()),
```

Comment:
I think I would leave it as Ephem because it is currently working version and changing it level db is easy